### ISSUE TITLE
artifact in the deblured image

### ISSUE THREAD

**Issue by scimg** [KupynOrest/DeblurGAN#87]
I run test.py with the default parameters and the downloaded latest_net_G.pth. I feed the algorithm with self-captured images and GoPro images. There are artifacts in the deblurred image, such as gridded artifacts with false colors. And the artifacts may relate to input size, as changing size setting will make it better. However, I still can not reproduce the impressive result presented in the paper. Would you please tell how to generate the results in your paper? Thanks you!

**Comment by chenyuZha** [comment]
@scimg I also ran the test with the latest .pth, I had same problem of artifact. I think that for the pre-trained model, the size of input image is 256x256, to get a better result, maybe you should just fine-tuning the model with a larger input size (512 or 1024).

**Comment by WangKewei1998** [comment]
Have you solved the problem yet? I also face the same problem when trying to run the test. I tried images from GoPro & my own images. Though it shows some effect but not as good as the paper shows.

**Comment by WangKewei1998** [comment]
@scimg

**Comment by scimg** [comment]
@WangKewei1998 Not yet.

**Comment by rimchang** [comment]
Also I cannot reproduce paper result.
I use below command.

python test.py --dataroot ./ --model test --dataset_mode single --learn_residual --loadSizeX 1280 --loadSizeY 720 --resize_or_crop test

below is evaluation result on GOPRO-test set. 
GOPR0869_11_00 Video's PSNR : 24.618, SSIM : 0.000, Count : 100
GOPR0862_11_00 Video's PSNR : 22.161, SSIM : 0.000, Count : 77
GOPR0410_11_00 Video's PSNR : 25.929, SSIM : 0.000, Count : 134
GOPR0384_11_05 Video's PSNR : 27.039, SSIM : 0.000, Count : 100
GOPR0385_11_01 Video's PSNR : 26.437, SSIM : 0.000, Count : 100
GOPR0881_11_01 Video's PSNR : 25.361, SSIM : 0.000, Count : 100
GOPR0854_11_00 Video's PSNR : 22.955, SSIM : 0.000, Count : 100
GOPR0396_11_00 Video's PSNR : 28.746, SSIM : 0.000, Count : 100
GOPR0384_11_00 Video's PSNR : 27.901, SSIM : 0.000, Count : 100
GOPR0868_11_00 Video's PSNR : 22.369, SSIM : 0.000, Count : 100
GOPR0871_11_00 Video's PSNR : 22.730, SSIM : 0.000, Count : 100
Total Video's PSNR : 25.199, SSIM : 0.000, Count : 1111

Have you solved the problem?
